website/docs/api/themes/theme-configuration.md

```diff
@@ -315,23 +347,31 @@ If you use docs with versioning, this special navbar item type that will render
 
 The user will be able to switch from one version to another, while staying on the same doc (as long as the doc id is constant across versions).
 
-```js {5-8} title="docusaurus.config.js"
+Accepted fields:
+
+| Field | Value | Explanation | Required | Default |
+| --- | --- | --- | --- | --- |
+| `position` | `'left'` / `'right'` | The side of the navbar this item should appear on. | No | `'left'` |
+| `dropdownItemsBefore` | Array of [link-like items](#navbar-dropdown) | Add additional dropdown items at the beginning of the dropdown. | No | `[]` |
```

Comment:
Also TS notation widely known so maybe we could have this?

[LinkLike](#navbar-dropdown)[]